conf/default.cfg

```diff
@@ -220,6 +220,13 @@ use = egg:oioswift#container_hierarchy
 # or
 #sentinel_hosts = 10.0.1.24:6012,10.0.1.27:6012,10.0.1.25:6012
 #sentinel_name = SENTINEL-master-1
+#
+# Redis parameters
+#redis_max_connections=2**31
+#redis_max_retry_on_timeout=False
+#redis_max_socket_connect_timeout=None
+#redis_max_socket_keepalive=False
+#redis_max_socket_timeout=None
```

Comment:
All keys are not prefixed with `max`: https://github.com/andymccurdy/redis-py/blob/f0516c9f7589557883a5eb23a44531066e275950/redis/connection.py#L886

Reply:
Good catch!